src/python/WMCore/ResourceControl/MySQL/ListThresholdsForSubmit.py

```diff
@@ -55,11 +55,14 @@ class ListThresholdsForSubmit(DBFormatter):
                   wmbs_location.id = job_count.location AND
                   wmbs_sub_types.id = job_count.subtype
                ORDER BY wmbs_sub_types.priority DESC"""
+
     seSql = """SELECT wl.site_name AS site_name,
                       wls.pnn AS pnn
```

Comment:
shouldn't this be  
```
wpnn.pnn as pnn
```
since previous code expect string not number?

Reply:
Nice catch! The worst part is that it doesn't crash anything, so we'd never know :-( I'll update it soon